build.sh

```diff
@@ -56,9 +56,9 @@ CXX=$CXXCOMP CC=$CCOMP cmake . -DNO_NATIVE_OPTIMIZATION=1 -DCMAKE_BUILD_TYPE=${C
 make
 
 # Package Server
-mkdir Server/Licenses
-cp Install/ThirdPartyLicenses/* Server/Licenses/
+cp -r Install/ThirdPartyLicenses/ Server/
```

Comment:
It copies the whole ThirdPartyLicenses folder to Server, so licenses do stay in a separate folder.